server/src/main/java/org/spine3/server/projection/ProjectionRepository.java

```diff
@@ -123,6 +150,13 @@ protected Storage createStorage(StorageFactory factory) {
     public void initStorage(StorageFactory factory) {
         super.initStorage(factory);
         setStatus(Status.STORAGE_ASSIGNED);
+
+        if(catchUpAfterStorageInit) {
+            if(log().isDebugEnabled()) {
```

Comment:
I see your point. I have added some parameters to let a developer know which `ProjectionRepository` is catching up.